From f115530c8c0c6e6ca1dfcba9a705b35aff7c6b93 Mon Sep 17 00:00:00 2001
From: Martin Kosek <mkosek@redhat.com>
Date: Thu, 28 Feb 2013 15:21:38 +0100
Subject: [PATCH 2/2] Fix includedir directive in krb5.conf template

We did not have the includedir directory with a trailing slash which made
rpm update add a redundant line.

https://fedorahosted.org/freeipa/ticket/3132
---
 install/share/krb5.conf.template | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/install/share/krb5.conf.template b/install/share/krb5.conf.template
index ed30b9e0fe37151c097d25e8b2e9fcf600a0a690..01e66881b0a38e342886727ec205ea9b7c057ad2 100644
--- a/install/share/krb5.conf.template
+++ b/install/share/krb5.conf.template
@@ -1,4 +1,4 @@
-includedir /var/lib/sss/pubconf/krb5.include.d
+includedir /var/lib/sss/pubconf/krb5.include.d/
 
 [logging]
  default = FILE:/var/log/krb5libs.log
-- 
1.8.1.2

